code/modules/clothing/head/helmet.dm

```diff
@@ -568,3 +568,84 @@
 	fire = 50
 	acid = 50
 	wound = 30
+
+/obj/item/clothing/head/helmet/durability/hit_reaction(mob/living/carbon/human/owner, atom/movable/hitby, attack_text = "the attack", final_block_chance = 0, damage = 0, attack_type = MELEE_ATTACK, damage_type = BRUTE)
+	take_damage(1, BRUTE, 0, 0)
+
+/obj/item/clothing/head/helmet/durability/watermelon
+	name = "Watermelon Helmet"
+	desc = "A helmet cut out from a watermelon. Might take a few hits, but don't expect it whitstand much."
+	icon_state = "watermelon"
+	inhand_icon_state = "watermelon"
+	flags_inv = HIDEEARS
+	dog_fashion = /datum/dog_fashion/head/watermelon
+	armor_type = /datum/armor/helmet_watermelon
+	max_integrity = 15
+
+/obj/item/clothing/head/helmet/durability/watermelon/fire_resist
+	resistance_flags = FIRE_PROOF
+	armor_type = /datum/armor/helmet_watermelon_fr
+
+/datum/armor/helmet_watermelon
+	melee = 15
+	bullet = 10
+	energy = 10
+	bomb = 10
+	fire = 0
+	acid = 25
+	wound = 5
+
+/datum/armor/helmet_watermelon_fr
+	melee = 15
+	bullet = 10
+	energy = 10
+	bomb = 10
+	fire = 15
+	acid = 30
+	wound = 5
+
+/obj/item/clothing/head/helmet/durability/holymelon
+	name = "Holymelon Helmet"
+	desc = "A helmet from a hollowed out holymelon. Might take a few hits, but don't expect it whitstand much."
+	icon_state = "holymelon"
+	inhand_icon_state = "holymelon"
+	flags_inv = HIDEEARS
+	dog_fashion = /datum/dog_fashion/head/holymelon
+	armor_type = /datum/armor/helmet_watermelon
```

Comment:
Does this just have the same armor as the watermelon? No side-grade or anything?

Reply:
I am yet to figure how to give it the ended effect, but *armour* is the same, yeah.

Reply:
donity done